src/Home/Home.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { map, curry } from "ramda";
+
+import FilterControls from "@/FilterControls/FilterControls"
+import MealCard from "@/MealCard/MealCard";
+import Overview from "@/Overview/Overview";
+
+
+const renderMealCards = (handler) => map((meal) => {
+	return <MealCard meal={ meal } key={ meal.id } handleUseIt={ handler.bind(null, meal.id) }/>
```

Comment:
of course I could have done this inline, but this notation feels to me like a really nice balance of terseness and expressiveness.

Reply:
I think there's a lot of art to how you lay things out in JSX, personally. :) Sometimes an expression looks better inline, sometimes better broken into multiple - it's part of what I like about it. The continual challenge of making something lovely and readable. :)